wiki/misconceptions.md

```diff
@@ -0,0 +1,187 @@
+# Misconceptions
+
+{title candidates: Misconceptions, Rebuttals, Common criticism}
+
+Misconceptions and common arguments against on-chain governance, coin voting and other aspects of Decred system.
+
+## Coin voting is plutocracy
+ 
+Counterarguments:
+
+- Company shareholder governance is also plutocracy, but nobody complains about that.
+- Voluntary: "plutocracy" is often used to mean non-voluntary (forced) participation, e.g. when you are born into a such society. You were not asked if you would like to participate. In public blockchains, on the opposite, no plutocracy happens to you until you voluntary join a given system.
+- Switching cost: "plutocracy" is often used in context where the switching cost is high. On the opposite, switching from one public blockchain to another is low (sell token A, buy token B). Low cost doesn't mean small losses due to price dynamics. It means it is low effort to perform the switch - to exchange tokens.
+
+Discussion:
+
+- https://medium.com/@richardred/what-is-on-chain-cryptocurrency-governance-is-it-plutocratic-bfb407ef6f1
+- https://medium.com/mosaic-network-blog/is-plutocratic-on-chain-governance-really-a-bad-thing-68132700205c
+
+## Ticket voting is not inclusive enough, tickets are expensive
+
+First, "expensive" for what? The amount of power one gets for a ticket is substantial. Ticket allows one to have a say in project's direction, voting on valid chain, consensus changes and budget decisions. This is not small. "Including" people with a few hundred bucks would be nice, but has lower priority than other things.
```

Comment:
One ticket is almost by definition 1/~40,960th of the decision-making power. There cannot be any more or less than the totality, the question of how many pieces to chop it into is secondary, and 40,960 is probably enough for now.